apps/package.json

```diff
@@ -279,54 +206,128 @@
     "@typescript-eslint/eslint-plugin": "^5.57.0",
     "@typescript-eslint/parser": "^5.57.0",
     "ace-builds": "^1.4.12",
+    "aws-sdk": "2.1231.0",
     "blockly": "10.3.1",
+    "bootstrap-sass": "^3.3.6",
+    "buffer": "^6.0.3",
+    "canvg": "gabelerner/canvg",
     "chart.js": "^4.4.0",
+    "classnames": "^2.3.2",
+    "codemirror": "5.5",
+    "codemirror-spell-checker": "^1.1.2",
     "crypto-browserify": "^3.12.0",
+    "csv-parse": "^4.4.6",
     "dapjs": "^2.3.0",
+    "data-collection": "^1.1.6",
     "details-element-polyfill": "https://github.com/javan/details-element-polyfill",
+    "fast-memoize": "2.0.2",
     "filesaver.js": "0.2.0",
+    "firebase": "^2.4.2",
+    "firmata": "^2.2.0",
     "focus-trap-react": "^10.1.1",
     "grunt-ts": "^6.0.0-beta.22",
+    "hammerjs": "^2.0.8",
+    "html2canvas": "^0.5.0-beta4",
     "htmlhint": "^1.1.4",
+    "immutable": "3.8.1",
     "is-hotkey": "^0.2.0",
+    "isolate-react": "^2.3.0",
+    "jquery": "1.12.1",
+    "jquery-ui": "^1.12.1",
+    "js-cookie": "^2.1.2",
+    "js-yaml": "^3.6.0",
+    "jsdom": "^21.1.1",
     "jshint": "^2.9.5",
+    "jsonic": "^0.3.0",
     "jszip": "3.10.1",
+    "lazysizes": "^4.0.0-rc1",
+    "loadable-components": "2.2.3",
+    "lodash": "^4.17.21",
     "memoize-one": "^5.1.1",
+    "messageformat": "2.3.0",
     "ml-knn": "^3.0.0",
+    "moment": "^2.29.4",
+    "newrelic": "^1.27.2",
     "object-fit-images": "^3.2.3",
     "pa11y-ci": "^2.3.0",
```

Comment:
devDependencies?

Reply:
My thinking was: while newrelic is a tool for devs, as far as I could tell (a little hard since I couldn't actually find the import/require for newrelic, all the accesses were to `window.newrelic`) its exclusively used at runtime vs from webpack or test code. You would know more, does that match your understanding?